Source/Setup.cpp

```diff
@@ -153,41 +164,48 @@ PeleC::variableSetUp()
 #endif
 
   // Set number of state variables and pointers to components
+  {
```

Comment:
Can we revert these changes that were an attempt to get around the cppcheck warnings and then Jon and I will look at how to fix the resulting warnings?

Reply:
Absolutely.